packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -194,6 +194,8 @@ describe('Checkout', () => {
 
     describe('data ::', () => {
         describe('serviceType ::', () => {
+            const fulfilmentTimeDropdownSelector = '[data-test-id="formfield-order-time-dropdown-select"]';
+            const tableIdentifierSelector = '[data-test-id="formfield-table-identifier-input"]';
```

Comment:
Separate variable initialisers from subsequent test definitions with a newline for readability, please:
```suggestion
            const tableIdentifierSelector = '[data-test-id="formfield-table-identifier-input"]';

```